tests/timezones/tests.py

```diff
@@ -121,21 +121,31 @@ def test_query_filter(self):
         dt2 = datetime.datetime(2011, 9, 1, 14, 20, 30)
         Event.objects.create(dt=dt1)
         Event.objects.create(dt=dt2)
-        self.assertEqual(Event.objects.filter(dt__gte=dt1).count(), 2)
-        self.assertEqual(Event.objects.filter(dt__gt=dt1).count(), 1)
-        self.assertEqual(Event.objects.filter(dt__gte=dt2).count(), 1)
-        self.assertEqual(Event.objects.filter(dt__gt=dt2).count(), 0)
+        test_values = (
```

Comment:
I think it's better to have just the input to the queries in the input to `subTest`, rather than the whole queries, so a tuple of `(lookup, datetime, count)`. Please consider this comment for similar cases.